docs/remotes.md

```diff
@@ -18,7 +18,7 @@ When running this command, you will be prompted first for the remote you want to
 
 ## `git: push`
 
-This command performs a simple `git push`.  In recent Git versions, this will result in a push to the tracking branch if it exists, else a remote branch with the same name as the local.
+This command will push current branch to the tracking branch. If the tracking branch is not set, you will be asked to configure it.
```

Comment:
Is there any case where a user would like to use push the branch to <remote>/<branch> without setting a tracking branch? Perhaps if a mistake is made when configuring tracking? In my opinion it's a really good default; and we can point to the `[t] configure tracking` action in the BRANCHES dashboard.